enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -1369,7 +1369,7 @@ func downloadLayer(ctx context.Context, layer ctr.Layer, destDir string) error {
 				return status.UnavailableErrorf("chown file: %s", err)
 			}
 		default:
-			return status.UnavailableErrorf("unsupported tar entry type %q", header.Typeflag)
+			log.CtxWarningf(ctx, "Ignoring unsupported tar header %q type %q in oci layer", header.Name, header.Typeflag)
```

Comment:
nit: I think these warnings will be spammy. Would it work to explicitly skip character and block devices?

Reply:
or alternatively just log at info level for now.